src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -748,4 +749,14 @@ public static void onCommandRegister(CommandDispatcher<CommandSource> dispatcher
         MinecraftForge.EVENT_BUS.post(evt);
         return evt;
     }
+
+    public static int onLivingBurnBySunlight(LivingEntity living, int duration)
```

Comment:
This return time has become pointless now if everything is handled internally. Should be `void`.

Reply:
The reason that I return the duration there is I think other users may need the updated value when they are making their custom entities.

Reply:
That doesn't make much sense, the modder could just call `getFlameTimer` to get the actual value or create their own version of this method that fires the event. Just return void here.

Reply:
Made the factory return void.